tafkars-lemmy/src/api_translation.rs

```diff
@@ -212,55 +161,17 @@ pub fn comment(state: &ResponseState, cv: CommentView) -> Comment {
     };
     Comment {
         data: CommentData {
-            total_awards_received: None,
-            approved_at_utc: None,
-            link_id: None,
-            author_flair_template_id: None,
-            likes: None,
-            saved: Some(false),
             id: Some(id.clone()),
-            gilded: Some(0),
-            archived: Some(false),
-            no_follow: None,
             author: Some(author),
-            can_mod_post: Some(false),
```

Comment:
Some of these fields shouldn't be removed: `Some(false)` is different from `None`. RedReader fails to parse the response like this.

Reply:
Changed back